app/plugin_api_test.go

```diff
@@ -220,6 +224,38 @@ func TestPluginAPIGetProfileImage(t *testing.T) {
 	require.Nil(t, data)
 }
 
+func TestPluginAPISetProfileImage(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	api := th.SetupPluginAPI()
+
+	// Create an 128 x 128 image
+	img := image.NewRGBA(image.Rect(0, 0, 128, 128))
+	// Draw a red dot at (2, 3)
+	img.Set(2, 3, color.RGBA{255, 0, 0, 255})
+	buf := new(bytes.Buffer)
+	err := png.Encode(buf, img)
```

Comment:
Please add `require.Nil(t, err)` after this, just to make sure.

Reply:
done